### ISSUE TITLE
Multiple comments for multiple plans

### ISSUE THREAD

**Issue by ChaosMarc** [dflook/terraform-github-actions#231]
### Suggestion

If there are multiple jobs that e.g. simultaneously run plans in different directories longer runnings job will just overwrite the comment of faster jobs instead of adding an additional comment. An option to toggle the behavior would be lovely 😍 

This is my example configuration with which I run into the problem:
```
jobs:
  dev_plan:
    name: Plan Terraform (DEV)
    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Plan Terraform (DEV)
        uses: dflook/terraform-plan@v1
        with:
          backend_config_file: environments/dev/backend.conf
          var_file: environments/dev/variables.tfvars

  common_plan:
    name: Plan Terraform (COMMON)
    runs-on: ubuntu-latest

    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Plan Terraform (COMMON)
        uses: dflook/terraform-plan@v1
        with:
          path: common
          backend_config_file: environments/common/backend.conf
          var_file: environments/common/variables.tfvars

```

**Comment by dflook** [comment]
Hello @ChaosMarc, it should be creating separate comments for each plan. 

What backend are you using? Can you share both of the backend blocks. What attributes are being set through the backend config file?

**Comment by ChaosMarc** [comment]
Thank you for the fast reply. Here are the configurations you asked for:
```
terraform {
  required_version = ">= 1.3.5"

  required_providers {
    azurerm = {
      source  = "hashicorp/azurerm"
      version = "3.32.0"
    }
  }

  backend "azurerm" {}
}

provider "azurerm" {
  features {}
}
```

environments/dev/backend.conf
```
resource_group_name  = "xxx-common"
storage_account_name = "xxxtfstate"
container_name       = "tfstate"
key                  = "dev.tfstate"
```

environments/common/backend.conf
```
resource_group_name  = "xxx-common"
storage_account_name = "xxxtfstate"
container_name       = "tfstate"
key                  = "common.tfstate"

```

**Comment by dflook** [comment]
Can you set the actions secret `ACTIONS_STEP_DEBUG` to `true` and run it again, then share the workflow log?

**Comment by ChaosMarc** [comment]
Here you go: [logs_93.zip](https://github.com/dflook/terraform-github-actions/files/10101184/logs_93.zip)

Thank you for your support!

**Comment by dflook** [comment]
Thanks. I can see the issue and it will be fixed in the next release.

**Comment by dflook** [comment]
This is fixed in v1.31.1

**Comment by ChaosMarc** [comment]
Works like a charm. Thank you very much for your super fast support :)

![image](https://user-images.githubusercontent.com/1343751/205046480-98ed55eb-c2c2-497f-9d28-d6a99eb91360.png)
